mappings/net/minecraft/client/render/model/json/ModelVariant.mapping

```diff
@@ -29,9 +29,9 @@ CLASS net/minecraft/class_813 net/minecraft/client/render/model/json/ModelVarian
 		METHOD method_67943 asModelBakeSettings ()Lnet/minecraft/class_3665;
 		METHOD method_67944 (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/kinds/App;
 			ARG 0 instance
-		METHOD method_67945 setRotationX (Lnet/minecraft/class_10821;)Lnet/minecraft/class_813$class_10803;
+		METHOD method_67945 withRotationX (Lnet/minecraft/class_10821;)Lnet/minecraft/class_813$class_10803;
 			ARG 1 amount
-		METHOD method_67946 setUVLock (Z)Lnet/minecraft/class_813$class_10803;
+		METHOD method_67946 withUVLock (Z)Lnet/minecraft/class_813$class_10803;
```

Comment:
Might be worth changing this to `withUvLock` when it's being renamed anyway? As well as as`ModelVariant.ModelState.withUVLock`